src/MapperMetadata.php

```diff
@@ -229,7 +229,8 @@ private function buildPropertyMapping(): void
                 null,
                 new CallbackTransformer($property),
                 $property,
-                false
+                false,
+                isPublic: true,
```

Comment:
because we're in presence of custom mapping, I think we should consider it as public, other wise `PropertyMapping::shouldIgnoreProperty()` will return true